feature_engine/datasets/data.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+import numpy as np
+
+
+def load_titanic(return_X_y_frame=False, predictors_only=False, raw=False):
+    df = pd.read_csv("https://www.openml.org/data/get_csv/16826755/phpMYEkMl")
+
+    if predictors_only:
+        df.drop(
+            columns=["name", "ticket", "home.dest", "boat", "body", "cabin"],
```

Comment:
This looks really good. 

I'd remove cabin from this list, because cabin is actually a good predictor, and try to add the functionality to extract the first cabin and (perhaps optional) extract the first letter from the cabin, because this is the one that indicates class, which correlates with survival.

Reply:
Actually I removed `cabin` because 1014 out of 1309 are null values. So in case of implementing what you say... how should I impute this variable with this amount of missing values? Probably a missing indicator with CategoricalImputer, because mode doesn't seem to be a good idea...

Reply:
makes sense, thank you!

Reply:
makes total sense. thank you